airflow/migrations/versions/5e3ec427fdd3_increase_length_of_email_and_username.py

```diff
@@ -46,9 +46,11 @@ def upgrade():
 
 def downgrade():
     """Revert length of email from 256 to 64 characters"""
```

Comment:
question ... 
what happens when this migration fails?  e.g. what if a user has been added whose username is greater than 64 characters?

perhaps once the length has been increased, we should not reduce it, even in a downgrade.  because i'm thinking that (1) leaving it as is isn't going to hurt anyone -- i.e. it's not a material issue -- and (2) changing it has a possibility of breaking the downgrade.

perhaps downgrades should be "pragmatic" in this way?

Reply:
Agree with you. MSSQL enforces this while other DBs don't enforce it.
I'll remove it and fix every other places we increased lengths of fields.
@ashb @jedcunningham @uranusjr Do you have contrary opinion on this?

Reply:
cool, just fyi i mentioned to @kaxil he seemed pretty lukewarm on the idea maybe wait for others to weigh in before making that change

Reply:
Yeah, a feature/code in the old version (that added that revision) might be depending on the "length" at that time.

So I think we should revert the chars length